integration-test/src/test/scala/io/phdata/clients/EmailClientImplIntegrationSpec.scala

```diff
@@ -27,7 +27,7 @@ class EmailClientImplIntegrationSpec extends FlatSpec {
     System.setProperty("mail.debug", "true")
 
     val htmlContent = templateEngine.layout(
-      "/templates/emails/welcome.mustache",
+      "common/src/main/templates/emails/welcome.mustache",
```

Comment:
Why the location change? Is there work to be done to package the image into the parcel?

Reply:
because "common/src/main/templates/emails/welcome.mustache" is the proper path in archway project.